core/shared/src/main/scala/matryoshka/Recursive.scala

```diff
@@ -165,6 +165,16 @@ trait Recursive[T] extends Based[T] {
       A =
     gcata[Cofree[H, ?], A](t)(distGHisto(g), f)
 
+  def gcataZygo[W[_]: Comonad, A, B]
```

Comment:
Indirectly, because of https://github.com/slamdata/matryoshka/pull/79/files/2641f7f624f4e15d3df1974b4532f874aac94021#diff-08bebdfb63dd73c8d3dab8adcf5d9777R73